apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/ServerInfoController.java

```diff
@@ -0,0 +1,190 @@
+        description = "Returns constants  information on current node in run-time",
+        tags = {"info"},
+        responses = {
+            @ApiResponse(responseCode = "200", description = "Successful execution",
+                content = @Content(mediaType = "application/json",
+                    schema = @Schema(implementation = BlockchainConstantsDto.class)))
+        }
+    )
+    @PermitAll
+    public Response blockchainConstants() {
+        log.debug("Started blockchain Constants");
+        ResponseBuilder response = ResponseBuilder.startTiming();
+        BlockchainConstantsDto dto = serverInfoService.getBlockchainConstants();
+        log.debug("blockchain Constants result : {}", dto);
+        return response.bind(dto).build();
+    }
+
+    @Path("/blockchain/state")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+        summary = "Returns status + additional data field information",
+        description = "Returns status + additional data field information for node",
+        tags = {"info"},
+        responses = {
+            @ApiResponse(responseCode = "200", description = "Successful execution",
+                content = @Content(mediaType = "application/json",
+                    schema = @Schema(implementation = BlockchainStateDto.class)))
+        }
+    )
+    @RolesAllowed("admin")
+    public Response blockchainState(
+        @Parameter(name = "includeCounts", description = "true for including additional data", allowEmptyValue = true)
+            @QueryParam("includeCounts") Boolean includeCounts,
+        @Parameter(description = "The admin password.", required = true) @QueryParam("adminPassword") String adminPassword
+        ) {
+        log.debug("Started blockchain State: \t includeCounts = {}", includeCounts);
+        ResponseBuilder response = ResponseBuilder.startTiming();
+        // that dto is BlockchainStatusDto + additional fields in BlockchainStateDto
+        BlockchainStateDto dto = serverInfoService.getBlockchainState(includeCounts);
+        log.debug("blockchain State result : {}", dto);
+        return response.bind(dto).build();
+    }
+
+    @Path("/blockchain/time")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+        summary = "Returns current Time information",
+        description = "Returns current Time information for node",
+        tags = {"info"},
+        responses = {
+            @ApiResponse(responseCode = "200", description = "Successful execution",
+                content = @Content(mediaType = "application/json",
+                    schema = @Schema(implementation = TimeDto.class)))
+        }
+    )
+    @PermitAll
+    public Response blockchainTime() {
+        log.debug("Started blockchain Time");
+        ResponseBuilder response = ResponseBuilder.startTiming();
+        TimeDto dto = serverInfoService.getTime();
+        log.debug("blockchain Constants result : {}", dto);
```

Comment:
get request logs may produce a lot of useless logs, set it to trace or remove if not needed.

Reply:
lowered to TRACE